src/catalog/src/system.rs

```diff
@@ -385,11 +388,14 @@ pub struct TableEntry {
     pub schema_name: String,
     pub table_name: String,
     pub table_id: TableId,
+    // None stands for mito engine
```

Comment:
Why isn't it an option?

Reply:
Sorry for the misleading comment. I'm removing the comment, and `engine` should be `String.`